src/main/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/JavadocLexer.g4

```diff
@@ -65,6 +67,16 @@ import java.util.*;
             _input.seek(_input.index() - 1);
       }
 
+      /**
+       * We need to create a different constructor in order to use our
+       * own implementation of the LexerATNSimulator. This is the
+       * reason for the unused 'crAwareConstructor' argument.
+       */
+      public JavadocLexer(CharStream input, boolean crAwareConstructor) {
+        super(input);
+        _interp = new CrAwareLexerSimulator(this, _ATN, _decisionToDFA, _sharedContextCache);
+      }
+
```

Comment:
We have to make a new constructor since ANTLR no longer supports the creation of abstract Lexers/ Parsers.